src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(';');
+
+  return styles.reduce((prev, item) => {
+    if (item.trim()) {
+      let [key, val] = item.split(':');
+
+      key = key.trim();
+      val = val.trim();
+      prev[key] = val;
+    }
+
+    return prev;
+  }, {});
```

Comment:
The reduce function processes each style declaration. However, it is important to note that the task does not require handling cases with multiple semicolons without any styles between them. The current implementation already handles this by checking if `item.trim()` is truthy before processing it, which is correct.